jenkins/internal-ci/generic/branch/release.groovy

```diff
@@ -125,15 +125,31 @@ pipeline {
         }
         
         stage ('Sign rpm') {
-            when { expression { false } }
+            when { expression { true } }
```

Comment:
We can remove this line if need to execute this stage each time.
CC - @shailesh-vaidya

Reply:
Removed this line from code